functions.php

```diff
@@ -771,6 +771,20 @@ function video_embed_html( $html, $url ) {
 }
 add_filter( 'embed_oembed_html', 'video_embed_html', 10, 3 );
 
+/*
+ * Add responsive container to YouTube embeds
+ */
+function embed_html( $html, $url ) {
+	if ( strpos( $url, 'facebook.com' ) !== false
+		|| strpos( $url, 'twitter.com' ) !== false
+		|| strpos( $url, 'instagram.com' ) !== false ) {
+		return '<div align="center">' . $html . '</div>';
```

Comment:
The `align` attribute is technically deprecated in HTML5; we were using it previously because we didn't have any other means of centering embeds.  Can we use a class instead that applies centering to its immediate child elem?  e.g.
```php
return '<div class="centered-embed">' . $html . '</div>';
```

```css
.centered-embed > * {
  margin-left: auto !important;
  margin-right: auto !important;
}
```